_includes/assets/js/indicatorModelHelpers.js

```diff
@@ -0,0 +1,539 @@
+(function() {
+  return {
+    UNIT_COLUMN: 'Units',
+    GEOCODE_COLUMN: 'GeoCode',
+    YEAR_COLUMN: 'Year',
+    VALUE_COLUMN: 'Value',
+    convertJsonFormat: function(data) {
+      var keys = Object.keys(data);
+      if (!keys.length) {
+        return [];
+      }
+
+      return data[keys[0]].map(function(item, index) {
+        return _.object(keys, keys.map(function(key) {
+          return data[key][index];
+        }));
+      });
+    },
```

Comment:
It is unclear what the purpose of this function is from the name and content.

1. Give the function a more meaningful name (`convertXToJson` or `convertJsonToX` depending on purpose) so it knows what it returns and what it expects as input.
2. `!key.length` could be changed to a more explicit `keys.length === 0` to imply "key array is empty".